src/main/java/build/buildfarm/common/StringUtils.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 The Bazel Authors. All rights reserved.
```

Comment:
Could just use an existing library for this instead of writing our own - org.apache.commons.lang3.StringUtils has this.

Reply:
thank you! way smarter!